jumpscale/servers/threebot/threebot.py

```diff
@@ -199,15 +199,18 @@ class Package:
     def __init__(self, path, default_domain, default_email, giturl=""):
         self.path = path
         self.giturl = giturl
-        self.config = self.load_config()
-        self.name = self.config["name"]
+        self._config = None
+        self.name = j.sals.fs.path_parts(f"{path}")[-1]
         self.nginx_config = NginxPackageConfig(self)
         self._module = None
         self.default_domain = default_domain
         self.default_email = default_email
 
     def load_config(self):
-        return toml.load(j.sals.fs.join_paths(self.path, "package.toml"))
+        path = j.sals.fs.join_paths(self.path, "package.toml")
+        if j.sals.fs.exists(path):
+            return toml.load(path)
+        return None
```

Comment:
keep it as it is, we need it to fail if the config is not exists maybe you can just check if the package.toml exists and raise an error if it is not

Reply:
but it's a bug because when raise error , i can't delete it .
the issue is how to delete corrupted package , so what if package.toml doesn't exist and this package is added before , should i can delete it

Reply:
we will not call load_config in the delete method so it will not be a problem